modules/test/playwright/pages/commerce/specificationFacets.page.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {CommerceLayoutsPage} from './layouts.page';
+
+export class SpecificationFacetsPage {
+	constructor(page) {
+		this.layoutsPage = new CommerceLayoutsPage(page);
+		this.page = page;
+		this.searchOptionsConfiguration = page.frameLocator(
```

Comment:
please use data-qa-id

Reply:
I don't know if it's possible to use data-qa-id attribute here